src/analog/m2kanalogin_impl.hpp

```diff
@@ -147,6 +151,9 @@ class M2kAnalogInImpl : public M2kAnalogIn, public libm2k::utils::DeviceIn
 	const double *getSamplesInterleaved(unsigned int nb_samples, bool processed = false);
 
 	double processSample(int16_t sample, unsigned int channel);
+
+	int convertVoltsToRawVerticalOffset(ANALOG_IN_CHANNEL channel, double vertOffset);
```

Comment:
Shouldn't these methods be `const`? It seems to me that they won't affect the internal state of the `m2k_analogin_impl` object. http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rconst-fct